v2/emailpassword/advanced-customizations/apis-override/usage.mdx

```diff
@@ -75,4 +77,82 @@ SuperTokens.init({
 
 - `originalImplementation` and `originalImplementationEmailVerification` are objects that contain apis that have the original implementation for this and the email verification recipe. They can be used in your custom apis as a way to use the SuperTokens' default behaviour.
 - In the above code snippet, we override the `signUpPOST` api of this recipe. This api will be called when the user clicks the sign up button on the frontend.
-- Likewise, we override the `verifyEmailPOST` api from the email verification recipe.
\ No newline at end of file
+- Likewise, we override the `verifyEmailPOST` api from the email verification recipe.
+
+
+## Method 2: Modify the API code
+
+If you need to change the implementation of the API at the code level you should use this method.
+
+:::important
+In this example we will be overriding the signIn API.
+:::
+
+### Step 1: Copy the API code from the SDK.
+
+We first need to find the implementation for signInPOST, this can be found [here](https://github.com/supertokens/supertokens-node/blob/master/lib/ts/recipe/emailpassword/api/implementation.ts#L95).
+
+### Step 2: Add the API code in the override config.
+Now in the override config instead of calling the original implementation like we did in [Method 1](#method-1-use-the-override-config) we can paste the signInPOST code we copied earlier and modify it.
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```jsx
+SuperTokens.init({
+    appInfo: {...},
+    supertokens: {...},
+    recipeList: [
+        EmailPassword.init({
+            // highlight-start
+            override: {
+                apis: (originalImplementation) => {
+                    return {
+                        ...originalImplementation,
+
+                        signInPOST: async (input) => {
+
+                            // we add the default implementation code we copied from the SDK and add custom logic.
+                            let newSignInPost = async ({ formFields, options }) => {
```

Comment:
Why have you made this a function.. just directly past the entire function body into the `signInPOST` function itself....